core/templates/domain/admin/admin-backend-api.service.spec.ts

```diff
@@ -536,10 +536,12 @@ describe('Admin backend api service', () => {
       username, method, category, languageCode
     ).then(successHandler, failHandler);
 
-    let req = httpTestingController.expectOne(
-      '/removecontributionrightshandler');
-    expect(req.request.method).toEqual('PUT');
-    expect(req.request.body).toEqual(payload);
+    Object.keys(
+      params).forEach((key) => (params[key] === null) && delete params[key]);
+    let query = new URLSearchParams(params);
+    let url = '/contributionrightshandler?' + query.toString();
+    let req = httpTestingController.expectOne(url);
```

Comment:
Prefer declaring as `const` if the variables are not being re-assigned. Here and below.

Reply:
Done.